### ISSUE TITLE
Minor formatting quirk in channels tab

### ISSUE THREAD

**Issue by millsdmb** [darkwallet/darkwallet#139]
![image](https://cloud.githubusercontent.com/assets/7471116/3254091/9c1bfed0-f1ec-11e3-9687-2c7952d8a1ea.png)

**Comment by caedesvvv** [comment]
can you describe the issue?

**Comment by millsdmb** [comment]
yeah, the channel box doesnt look aesthetically correct, the line is off.

**Comment by leafcutterant** [comment]
Well, you can't have a box that extends all over because of one long string. Maybe you could have more condensed text type and make it not end with an ellipsis (...) but simply leave the text flow as far as it can, to maximize the use of the space. I mean:

![untitled-1](https://cloud.githubusercontent.com/assets/7106231/3260141/dd7c40fc-f24a-11e3-9f91-5fc3281374c7.png)

And/or you could simply get rid of the hash value and display only the channel name.

**Comment by caedesvvv** [comment]
We will show just hash / name and add a way to toggle between the two.

Will be showing the hash, clicking the toggle will show the names for some secs.
